packages/core/src/lib/TwaGenerator.ts

```diff
@@ -194,6 +194,13 @@ export class TwaGenerator {
       throw new Error(
           `Failed to download icon ${iconUrl}. Responded with status ${response.status}`);
     }
+
+    const contentType = response.headers.get('content-type');
+    if (!contentType?.startsWith('image/')) {
```

Comment:
Can Android handle any type of image (can it do SVGs?)?

Reply:
The image that is read here is converted to PNG by `jimp` when saved. To your point, `jimp` doesn't support reading SVGs. So, we should reject it for now and create an issue to add support for enabling reading SVGs.

Reply:
Throwing a specific error for image/svg. I also created https://github.com/GoogleChromeLabs/llama-pack/issues/103